cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -257,11 +261,42 @@ func createConfigAPIEntry(c config.Config, apis api.APIs, project project.Projec
 			return persistence.DeleteEntry{}, fmt.Errorf("resolved name parameter is no string")
 		}
 		scopeValue = nameOfRefCfgStr
+
+		if apis[c.Coordinate.Type].ID == api.KeyUserActionsWeb {
+			// while generating the delete file (which happens offline) we are not able
+			// to resolve that parameter, so we delete it
+			delete(c.Parameters, config.ScopeParameter)
+			props, resolvErr := c.ResolveParameterValues(entities.New())
```

Comment:
won't this also fail if the config has any other reference parameter defined?

Reply:
of course 🤦